test_fixtures/align_content_space_around_wrapped.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <script src="../scripts/gentest/test_helper.js"></script>
+  <link rel="stylesheet" type="text/css" href="../scripts/gentest/test_base_style.css">
+  <title>
+    Test description
```

Comment:
Is this supposed to be replaced by an actual description or does this mean "this is a description of a test case"?

Reply:
Possibly, but none of the other tests have one. I also think you'd want more than the one-line description that would fit in the `title` element if you were going to have a description.